src/index.js

```diff
@@ -86,6 +86,26 @@ class Keyv extends EventEmitter {
 			.then(() => true);
 	}
 
+	has(key) {
+		key = this._getKeyPrefix(key);
+		const store = this.opts.store;
+
+		return Promise.resolve()
+			.then(() => {
+				return store.has(key);
+			});
+	}
+
+	isUndefined(key) {
```

Comment:
I'm not sure an `.isUndefined()` method is necessary, it can be done outside of Keyv once `.has()` is implemented.